src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -232,6 +245,32 @@ public void assertMatchingGenotypesFromTileDB(File input, File expected, Locatab
         runGenotypeGVCFSAndAssertSomething(genomicsDBUri, expected, NO_EXTRA_ARGS, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes, reference);
     }
 
+    @Test(dataProvider = "getGVCFsForGenomicsDBOverMultipleIntervals")
+    public void testGenotypeGVCFsMultiIntervalGDBQuery(File input, File expected, List<Locatable> intervals, String reference) throws IOException {
```

Comment:
If you really expect this test to be quick and only take 0.06 seconds we should probably have a contingency plan in place so this fails gracefully on travis (my fear is of a world where this fails and the signal is travis times out due to inactivity). You could add to this test a testNG timeout option ` timeOut = 1000` which would be 1000 ms or some more reasonable gross overestimation of the runtime that we expect to fail if this branch didin't work and use that.

Reply:
Whoops! Looks like I gave you a poor choice of number. I forgot to account for the runtime units being in terms of minutes. Sorry about that...